Source/CarthageKit/Git.swift

```diff
@@ -167,6 +167,27 @@ public func launchGitTask(
 	let taskDescription = Task("/usr/bin/env", arguments: [ "git" ] + arguments, workingDirectoryPath: repositoryFileURL?.path, environment: updatedEnvironment)
 
 	return taskDescription.launch(standardInput: standardInput)
+		.filter { taskEvent in
+			switch taskEvent {
+			case .success(_):
+				return true
+			case .launch(_):
+				return true
+			case let .standardOutput(data):
+				return data.count > 0
+			case let .standardError(data):
+				return data.count > 0
+			}
+		}
+		.on(event: { event in
+			guard let taskEvent = event.value else {
+				return
+			}
+			if case .success(_) = taskEvent {
+				return
+			}
+			print(taskEvent)
+		})
 		.ignoreTaskData()
```

Comment:
This is the issue here. Instead of ignoring the task data, it needs to be sent through. (The `standardOutput` and `standardError` events should bubble up to be printed. You can search for where this is done for `xcodebuild` builds.)